src/Halogen/HTML/Events.purs

```diff
@@ -100,6 +108,17 @@ onScroll = handler (EventType "scroll")
 onChange :: forall r i. (Event -> Maybe i) -> IProp (onChange :: Event | r) i
 onChange = handler ET.change
 
+onFileUpload :: forall r i t. Unfoldable t => Monoid (t File)
+             => (Effect (t File) -> Maybe i) -> IProp (onFileUpload :: Event | r) i
```

Comment:
@googleson78 there's no such HTML property or attribute as `files` so we can't add that here either. The others have `value` or `checked` as those are standard properties for inputs.

It does mean you can put `onFileUplaod` on anything with `onChange` (after my other comment is resolved), but since it's filtered on event type, it will just do nothing if used inappropriately.

Reply:
Can I resolve this?

Reply:
I think you still need to add the "right" property, so `onChange`/`onInput` instead of `onFileUpload`. I'm not sure which one is appropriate in this situation though.

Reply:
Can you please "right" that out?

Reply:
```purescript
onFileUpload :: forall r i t. Unfoldable t => Monoid (t File)
             => (t File -> Maybe i) -> IProp (onFileUpload :: Event | r) i
```
->
```purescript
onFileUpload :: forall r i t. Unfoldable t => Monoid (t File)
             => (t File -> Maybe i) -> IProp (onChange :: Event | r) i
```

I think this is what needs to happen

Reply:
Thanks

Reply:
Ok changed that.
Are there any docs that explain what is actually happening with using onChange vs onFileUpload in the type?